theme/less/_body-wrapper.less

```diff
@@ -12,6 +12,30 @@ body > div
   padding: 20px 30px;
 }
 
+@font-face 
+{
+  font-family: "Source Sans Pro";
+  font-weight: 400;
+  font-style: normal;
+  src: local("Source Sans Pro Regular"), local("SourceSansPro-Regular"), url("/fonts/SourceSansPro-Regular.woff2") format("woff2");
+}
+
+@font-face 
+{
+  font-family: "Source Sans Pro";
+  font-weight: 700;
+  font-style: bold;
```

Comment:
Not really sure if we need to specify that the font-size is bold, in general thinking we can ignore `font-style ` rule and use `font-weight: 700;` to resolve the one we need.